packages/jsii-pacmak/lib/targets/kotlin/kdocbuilder.ts

```diff
@@ -0,0 +1,46 @@
+import { CodeMaker } from 'codemaker'
+
+export class KDocBuilder {
+  private readonly chunks: string[] = [];
+
+  public push(
+    chunk: string | undefined,
+    prefix: string | undefined = undefined
+  ) {
+    if (!chunk || chunk.length === 0) {
```

Comment:
`!chunk` actually already covers `chunk === ''` which is another way to say `chunk.length === 0`. The empty string is false-y in Javascript!